kotlinx-coroutines-core/common/src/Dispatchers.common.kt

```diff
@@ -64,7 +64,7 @@ public expect object Dispatchers {
      * println("Done")
      * ```
      * Can print both "1 2 3" and "1 3 2". This is an implementation detail that can be changed.
-     * However, it is guaranteed that "Done" will be printed only when both `withContext` calls are completed.
+     * However, it is guaranteed that "Done" will only be printed once the `withContext` call is completed.
```

Comment:
```suggestion
     * However, it is guaranteed that "Done" will only be printed once both the `withContext` call and the inner `launch` are completed.
```
I think this reflects the point better: the order is not guaranteed, but `Dispatchers.Unconfined` will finish doing its thing by the time you leave the outermost invocation.